No-cache get_cutout() feature

lrodri29 opened the pull request:
Adds the option to get_cutout without using cache. 
Defaults to no_cache = False. 

Use:
```python
boss = BossRemote()
volume = boss.get_cutout(
    boss.get_channel("chan", "coll", "expp"), 1,
    [10000, 10500], [10000, 10500], [500, 550], no_cache = True,
)
```

movestill commented on intern/service/boss/baseversion.py:
```diff
@@ -147,7 +147,7 @@ def build_metadata_url(self, resource, url_prefix, key, value=None):
 
     def build_cutout_url(
-        self, resource, url_prefix, resolution, x_range, y_range, z_range, time_range=None, id_list=[]):
+        self, resource, url_prefix, resolution, x_range, y_range, z_range, no_cache, time_range=None, id_list=[]):
```
Move `no_cache` to the end and give it a default value of False.

movestill commented:
Made the no_cache parameter explicit so it's not invisible to potential users. Please do the same to the `boss/remote.py`. Also add it to the docstrings.

lrodri29 commented:
Tim, there is no `get_cutout` function in `remote/boss/remote.py` and I shouldn't add the explicit `no_cache=False` key arg to remote/remote.py since, as we had discussed, it is not necessarily relevant to every remote. 

I added the doc strings and created a `get_cutout` function in `remote/boss/remote.py` so that it is easy to find. I also updated the documentation html to include the no_cache option.
